### ISSUE TITLE
Some actual working examples

### ISSUE THREAD

**Issue by OFark** [SamProf/MatBlazor#883]
This is a feature of support rather than the actual app. I haven't gotten as far as the app as I'm giving up trying to use it. I'm sure it'll probably work fine, but I cannot work out how to use it from the examples and instructions or lack thereof. 

- Navigation items in the Header bar? How? I can't find any examples, and whatever I try, is either in the wrong font, the wrong colour, hidden or partly hidden. I don't know how MatNavMenu goes in MatAppBar, or if it even does?
- MatDrawerContainer, I can't get this to work, doesn't show anything. Is this supposed to inside MatAppBarContainer, or after it? The examples show page content in both so I'm assuming I'm supposed to nest MatDrawerContainer inside MatAppbarContent? I don't know because I can't see an example of it anywhere.
- MatAppBarContainer creates a burger menu, no idea how to bind anything to clicking that, as far as I can tell it does nothing.

I have some other suggestions that go along with what I've come to expect from frameworks like this:
- Some sort of normalisation, imagine the look on my face when Body element hasn't had its padding removed and my menu wasn't at the top and full width. 
- Your theme picker shows examples of using "MatThemeColors.Orange._500.Value", yet the theme picker/chart doesn't tell me what the colours are called, it could do with generating a code sample based on the colours I've picked.
- A starting point, the installation instructions are great, but it's clear to me your using custom controls to create a hierarchy, but nowhere is that complete hierarchy explained. An example of that is my Navigation issues outlined above. It seems, from the examples given, that page content is in both MatAppBarContent and MatDrawerContent. But I cannot see any instruction on how to use these two items together as you have on the example website. Does MatAppBarContainer wrap MatDrawerContainer? or the other way round? Have a look at [https://getmdl.io/](https://getmdl.io/) it is useless for Blazor but it gives a great starting point, and has great examples of usage.

**Comment by Christian-Oleson** [comment]
@OFark ,
Have you tried some examples on MatBlazor.com and followed the setup? Those are working, and so if there are issues with the setup docs, please let us know where you're stumbling. Otherwise, I'd love to see a PR from you to _improve_ things, but I'm not certain you've gotten through the setup far enough to be able to do so.

**Comment by OFark** [comment]
You are correct I haven't. The samples either don't work, or aren't covering a full usage scenario. MudBlazor is another example, of what's going on here, but they've opted to add sample code on the page that you can clearly see how the code is supposed to be used. The problem is with the how to use, rather than the project, there's no request to pull, the help needs more guidance on how to construct a hello world scenario. My best example of this is the navbar, it's a starting point for most apps. It has a menu that does nothing, and nowhere, that I could find, gave an example of how to tie this up to a drawer. If that menu button isn't an option, then neither is the drawer. Or if it is optional, let's see examples of both with and without.

**Comment by enkodellc** [comment]
@OFark I understand your frustration but this is basically how most every open-source project I have used is like. If you need support and detailed examples you might be better off with a paid / enterprise tool set. Either that I recommend you learn and contribute to the open-source repos, that is where you will learn more as well. Just my 2 cents.
